components/style/gecko/media_queries.rs

```diff
@@ -530,17 +616,19 @@ impl Expression {
                     Ok((f, r)) => {
                         feature = f;
                         range = r;
-                    }
-                    Err(()) => return Err(SelectorParseError::UnexpectedIdent(ident.clone()).into()),
+                    },
+                    Err(()) => {
+                        return Err(StyleParseError::MediaQueryExpectedFeatureName.into())
```

Comment:
We should pass the ident that was not recognized as a parameter of this error variant.